README.md

```diff
@@ -1,2 +1,42 @@
-# group-Jessica
-Household Manager Web Application using C#
+# Household Pro
+
+## Description
+Household Manager Web Application using C# and ASP.NET Core 6 via MVC created with the intent for 5 entry-level programmers to practice building an application from scratch utilizing Agile development and version-control via branching in GitHub.
+
+Household Pro allows individuals that live in a multi-person unit to collect, organize, and assign all chores and assignments involved in keeping up with home organization and cleanliness efficiently and faily.
+
+## Contributors
+- [**Alli**](https://github.com/alliology934)
+- [**Erica**](Add your link here like I have above.)
+- [**Michele**](Add your link here like I have above.)
+- [**Roxanna**](Add your link here like I have above.)
+- [**Spencer**](Add your link here like I have above.)
+
+## Features
+- Manipulation of database content through CRUD operations.
+- Authentication and authorization via Identity framework.
+- Clear and efficient method for choosing icons by jQuery implementation and icon API utilization.
+- Modern UI through intentional usage of Bootstrap 5, Sass, and theme selection feature.
+- Informative and useful display of data through sortable and pageable tables powered by Syncfusion UI components.
+
+## Technology Stack
+- **Language:** C#
+- **Framework:** .NET 6.0
+- **Template Engine:** ASP.NET Core MVC
+- **Database Engine:** SQL Express
+- **Other Libraries or Components:**
+  - Entity Framework
+  - Bootstrap 5
+  - Sass
+  - Syncfusion
+  - Identity
+  - Icon API
+  - Font Awesome
+
+## Instructions
+1. Fork this repository
+2. Clone your forked repository
+3. **TODO:** Stuff about downloading packages for the app to work
+4. **TODO:** Stuff about hooking up their local SQL Database
```

Comment:
4.  Through the Package Manager type "Update-Database" to initialize and migrate the seeded database with its tables to SQL Express.

Reply:
This is exactly what I needed, thanks! Changes made, and I also added a step before this to "Ensure Microsoft SQL Express is installed on your local machine."  I hope that is true. Won't hurt though at the very least, haha.